src/org/labkey/test/TestFileUtils.java

```diff
@@ -118,6 +120,15 @@ public static String getFileContents(Path path)
         }
     }
 
+    public static int getFileRowCount(final File file) throws IOException
+    {
+        BufferedReader reader = new BufferedReader(new FileReader(file));
+        int lines = 0;
+        while (reader.readLine() != null) lines++;
+        reader.close();
+        return lines;
```

Comment:
Use try-with-resources to ensure the reader is closed even if an exception is thrown.
```suggestion
        try (BufferedReader reader = new BufferedReader(new FileReader(file)))
        {
            int lines = 0;
            while (reader.readLine() != null) lines++;
            return lines;
        }
```